### PR TITLE
Feature/api create user show

### PR DESCRIPTION
**Pull request by mtakac2**
I've cherry picked only commits dealing with user show endpoint from feature/api-create-user-show-endpoint branch.

### CODE REVIEW

File: lib/api/v3/root.rb [hunk 1]
```diff
@@ -35,11 +35,12 @@ module API
...
       version 'v3', using: :path
-
-      mount ::API::V3::Attachments::AttachmentsAPI
-      mount ::API::V3::WorkPackages::WorkPackagesAPI
-      mount ::API::V3::Queries::QueriesAPI
       mount ::API::V3::Activities::ActivitiesAPI
+      mount API::V3::Attachments::AttachmentsAPI
+      mount API::V3::Queries::QueriesAPI
+      mount ::API::V3::Users::UsersAPI
+      mount API::V3::WorkPackages::WorkPackagesAPI
```
Comment (myabc) [inline lib/api/v3/root.rb]: @mtakac why did you use absolute namespacing for only of these classes?

Comment (mtakac2) [inline lib/api/v3/root.rb]: @myabc that was not my intent. I'll push commit with correction.

**Comment by myabc** [comment]
Specs fail.

**Comment by tilt** [comment]
I'll rewire the tabs now, @mtakac Is there a ticket for this?

**Comment by mtakac2** [comment]
@tilt no. open one.

**Pull request merged**